### ISSUE TITLE
bad gpg signature of envconsul_0.6.1_SHA256SUMS

### ISSUE THREAD

**Issue by blalor** [hashicorp/envconsul#110]
Just downloaded 0.6.1 from releases.hashicorp; I'm getting:
```
> gpg --verify envconsul_0.6.1_SHA256SUMS{.sig,}
gpg: Signature made Fri Feb 19 13:54:54 2016 EST using RSA key ID 348FFC4C
gpg: BAD signature from "HashiCorp Security <security@hashicorp.com>"
```

The downloaded files are attached: [envconsul_failed_sig.zip](https://github.com/hashicorp/envconsul/files/317187/envconsul_failed_sig.zip)

steps to reproduce:
```
ENVCONSUL_VERSION="0.6.1"
curl -sfSLO https://releases.hashicorp.com/envconsul/${ENVCONSUL_VERSION}/envconsul_${ENVCONSUL_VERSION}_linux_amd64.zip
curl -sfSLO https://releases.hashicorp.com/envconsul/${ENVCONSUL_VERSION}/envconsul_${ENVCONSUL_VERSION}_SHA256SUMS
curl -sfSLO https://releases.hashicorp.com/envconsul/${ENVCONSUL_VERSION}/envconsul_${ENVCONSUL_VERSION}_SHA256SUMS.sig
gpg --verify envconsul_${ENVCONSUL_VERSION}_SHA256SUMS{.sig,}
```

**Comment by sethvargo** [comment]
Hi @blalor 

I just checked, and I'm unable to reproduce this on releases.hashicorp.com downloads, but I _can_ reproduce it on your downloads. Can you try re-downloading the files please?

**Comment by blalor** [comment]
Yeah, I can't verify right now, either. Weird.

**Comment by sethvargo** [comment]
@blalor I'm going to chalk it up to corrupted file download. If it happens again, please open a new issue and we'll definitely investigate. Sorry!

**Comment by blalor** [comment]
No worries, thanks for looking into it.  Thing is, it was repeatable at the time.  I’m building AMIs with Packer (#hashilove) and had a bunch of builds running in parallel that all failed at the same time with that error (because I do validate the binaries when installing them).  Then I was able to reproduce it on my laptop, on a completely different network.
